frontend/src/model/beta/reviews/ReviewDisplay.tsx

```diff
@@ -35,9 +36,7 @@ export default function ReviewDisplay({ review }: ReviewDisplayProps) {
               <HourglassEmpty color='warning' fontSize='small' />
             )}
             <Typography variant='caption'>
-              <Box component='span' fontWeight='bold'>
-                {reviewResponse.user.split(':')[isAccepted ? 1 : 0]}
-              </Box>
+              <UserDisplay entityId={reviewResponse.user.split(':')[1]} />
```

Comment:
Why has the logic here changed from
```
reviewResponse.user.split(':')[isAccepted ? 1 : 0]
```
to
```
reviewResponse.user.split(':')[1]
```

Reply:
This is a user entity, of the form `user:name`.  Here we always want to display the latter part of it.  This is to fix a prior bug.